### ISSUE TITLE
Email reminders for claimed tasks

### ISSUE THREAD

**Issue by Paulina-Rosetta** [TheRosettaFoundation/SOLAS-Match#1202]
How difficult would it be to add notification emails that remind users of tasks they've claimed?
AFAIK there's only one reminder after you claim a task and this is sent to users who haven't uploaded  just after the deadline passes. There are no previous or subsequent reminders.

I would suggest to send one reminder a week before the task is due, one after the deadline passes and one very urgent one a week after the deadline has passed. This would save Alba a lot of emailing. Also, volunteers who claim tasks with far-away deadlines tend to forget all about them.

Any thoughts?

**Comment by alanbarrett** [comment]
I will look at that... it may not be very easy as the backend is harder to
change... but I will look when I get a chance.

    Alan.

**Comment by alanbarrett** [comment]
Could you send me the text for the two e-mails... (week before & week after the deadline)... here is the existing wording for the e-mail at the time of the deadline...

    The "{{TASK_TYPE}}" task for "{{TASK_TITLE}}" from {{SOURCE_LANGUAGE}} into {{TARGET_LANGUAGE}} has passed its deadline. 
    As the task deadline has passed, please ensure to upload your output file. 
    If you have any questions about the deadline, please contact <a href="info@trommons.org"> info@trommons.org. </a>

**Comment by alanbarrett** [comment]
Also I need the subject of the two e-mails. The existing Subject is...
trommons.org: Task Deadline Passed

**Comment by Paulina-Rosetta** [comment]
I'll have these ready by tomorrow

**Comment by Paulina-Rosetta** [comment]
Week before deadline:

Subject: Trommons Task Deadline Reminder
Thank you for claiming a volunteering task on Trommons!
You have one week to complete the "{{TASK_TYPE}}" task for "{{TASK_TITLE}}" from {{SOURCE_LANGUAGE}} into {{TARGET_LANGUAGE}}. 
Please ensure to upload your file before the deadline passes. 
If you have any questions about the deadline, please contact <a href="info@trommons.org"> info@trommons.org.

Week after deadline:

Subject: Trommons Task Overdue
Thank you for claiming a volunteering task on Trommons!
This is to remind you that the "{{TASK_TYPE}}" task for "{{TASK_TITLE}}" from {{SOURCE_LANGUAGE}} into {{TARGET_LANGUAGE}} is overdue.
Please ensure to upload your file as soon as possible. 
If you have any questions about this task, please contact <a href="info@trommons.org"> info@trommons.org.

**Comment by alanbarrett** [comment]
This is now implemented on the live server. I have tested pretty well on the dev server and all should be OK, but we will know more when reminders go out overnight.

Note you will not get notification that the e-mails have been sent. But they will go, as agreed, 1 week before and one week after the deadline. (The existing e-mail at the deadline will continue to be sent.)

Note, you will have noticed warning e-mails sent out because I had stopped the backend .

    Alan.
